### ISSUE TITLE
Keeping it science domain agnostic

### ISSUE THREAD

**Issue by Zarquan** [ivoa-std/ExecutionPlanner#43]
Keeping the specification science domain agnostic is important, because it will have implications for funding. Many of our funding bodies are looking to build on work done in individual science domains to develop global cloud compute platforms
that can be used by all science domains.

To qualify for future national and international funding I think the IVOA needs to start thinking outside the box and look at how our work could be used by other science domains.

The EU commission's work on developing the European Open Science Cloud (EOSC) as an international cross-domain science platform for all researchers is a good example. The EOSC Futures program is looking beyond individual science domains at solutions that work across multiple domains,

There is no reason why the core ExecutionPlanner specification specification needs to hav any astronomy specific details in it.
Its main function is orchestrating the execution steps for running Docker containers on Kubernetes and Slurm platforms. Cloud computing is a generic problem, so our solutions should be generic too.

I would like to position ExecutionPlanner at the same level as Rucio, and Reana. Both of which are developed by CERN, but neither of them has any particle-physics specific aspects to them. Which means they can be promoted as generic tools that any science domain can use.

TL;DR; - if we keep ExecutionPlanner science domain neutral, it makes it much more useful to many more people, and it will be easier to apply for funding from national and international funding agencies.
